docs/reference/transform/apis/start-transform.asciidoc

```diff
@@ -56,7 +57,16 @@ required privileges on the source and destination indices, the
 
 `<transform_id>`::
 (Required, string)
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=transform-id]
+Identifier for the {transform}.
```

Comment:
I wondered, but now I get what you are changing. You don't want the naming rules, as they only matter for creation?

However, wouldn't it make more sense to reverse the logic? I mean using `Identifier for the {transform}.` as common description and adding the naming rules only for create?

(Having that said, the include is longer than the content)

Reply:
Yes, that was the intention. Good point about flipping that. I've generally been trying to improve the text as I move it to the elasticsearch-specification files, and not worrying as much about replicating those improvements in the current docs, but that's a good idea.